package.json

```diff
@@ -47,8 +49,13 @@
     "http2-push-manifest": "^1.0.0",
     "lighthouse-ci": "https://github.com/ebidel/lighthouse-ci",
     "load-grunt-tasks": "^4.0.0",
+    "rollup": "^1.9.0",
+    "rollup-plugin-commonjs": "^9.3.4",
```

Comment:
what CJS modules are being used in the client? If lit-element is the only dependency, there should be any.

Reply:
I think `date-fns` is being used. It's called out in the rollup.config.

Reply:
CJS & date-fns: my bad, forgot to delete after testing the set up.

scripts: (from what I see) the `start` script only brings up the server, but nothing for frontend code refresh. So if I want to get auto compile when developing, I need to have one command window running start_server.sh, and another one running watch-component. (The gulp and grunt both aren't watched, I guess we used to just run it manually after editing?)
Is this the part you are talking about?